lib/instrumental.js

```diff
@@ -229,6 +240,12 @@ exports.init = function instrumental_init(startup_time, config, events) {
       recordCounterRates = config.instrumental.recordCounterRates;
     }
 
+    if(typeof(config.instrumental.metricPrefix) == 'undefined'){
```

Comment:
Every place else uses ==, I changed all of the configuration inspection to be strict.  I do not think this will break anyone's current config in production.